pkgs/development/python-modules/django-json-widget/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  django,
+  pythonOlder,
+  setuptools,
+  python,
+}:
+
+buildPythonPackage rec {
+  pname = "django-json-widget";
+  version = "2.0.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "jmrivas86";
+    repo = "django-json-widget";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-GY6rYY//n8kkWCJZk6OY+EOBv62ocNovNmE/ai8VCn4=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ django ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    ${python.interpreter} runtests.py
+
+    runHook postCheck
+  '';
```

Strong preference for running django tests through pytest.

```suggestion
  nativeCheckInputs = [
    pytest-django
    pytestCheckHook
  ];
  
  preCheck = ''
    export DJANGO_SETTINGS_MODULE=tests.settings
  '';
```

---

Thank you that worked :+1: